include/abe/fame.h

```diff
@@ -131,7 +132,8 @@ void cfe_fame_sec_key_free(cfe_fame_sec_key *sk);
 /**
  * Generates a master secret key and a public key.
  *
- * @param pk A pointer to an initialized cfe_fame_pub_key struct, the public key will be saved here
+ * @param pk A pointer to a cfe_fame_pub_key struct, the public key will be saved here;
+ * pk does not need to be called by any initialization function
```

Comment:
`pk does not need to be called by...` isn't good (it is a pointer to a struct, so it can't be called). Change it to `pk does not need to be manually initialized by...`.

Reply:
Ok, done.